### PR TITLE
chore(deps): add pgocaml

### PR DESCRIPTION
**Pull request by medson10**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,8 +1,10 @@
-DATABASE_URL=
+PGHOST=
+PGPORT=
+PGUSER=
+PGDATABASE=
+PGPASSWORD=
+COMMENT_SRC_LOC=
```
Comment (fakenickels) [inline .env.example:8]: what is this one for?

Comment (medson10) [reply .env.example:8]: pgocaml, the env vars it uses, this last one is for log somethings that's happening when the queries are called

File: schema/User.re [hunk 2]
```diff
@@ -13,30 +15,37 @@ type t = {
...
 
-module ModelConfig = {
-  type t = user;
-  let table = "app_users";
-  let parseRow = row => {
-    id: row[0],
-    email: row[1],
-    role: row[2] == "ADMIN" ? Admin : User,
+let parseRole = roleString =>
+  switch (roleString) {
```
Comment (fakenickels) [inline schema/User.re:19]: why role is `option(string)`?

Comment (medson10) [reply schema/User.re:19]: something wrong on the initial schema, gonna fix on another pr, this is not the only one wrong

**Review by fakenickels — APPROVED** [review]

File: .env.example (hunk 1 above)
Comment (fakenickels) [reply .env.example:8]: aah got it, would be cool to add as a comment as this is pretty implicit

**Comment by fakenickels** [comment]
Looking good 👍

File: .env.example (hunk 1 above)
Comment (medson10) [reply .env.example:8]: indeed, gonna add

**Pull request merged**